tests/webpack-test/configCases/asset-modules/build-http/webpack.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 	mode: "development",
 	experiments: {
 		buildHttp: {
-			allowedUris: [() => true],
+			allowedUris: ['https://'],
```

Comment:
do not currently support function callback, only regex and string pattern

Reply:
Let's add a additional test in rspack-test-tools, usually we won't change test inside webpack-test